packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -91,7 +95,7 @@ class SliderTheme extends InheritedWidget {
 ///
 /// See also:
 ///
-///  * [Slider], a Material Design slider widget.
+///  * [Slider], which includes a field defined by this enum.
```

Comment:
I don't think it includes such a field. Linking to Slider here (the original doc) is OK.